src/docs/truffle/reference/configuration.md

```diff
@@ -51,6 +51,138 @@ Any of the following solutions will remedy this issue:
 
 ## General options
 
+### solidityLog
+
+Like JavaScript's `console.log`, you can utilize the same concept to print
+state of your contract during execution. This log will appear when you use
+`truffle test` or `truffle develop`. To use it, import
+`truffle/console.sol` into your Solidity contract and use it like you would in
+JavaScript. See the [Nodejs' util.format()
+documentation](https://nodejs.org/api/util.html#utilformatformat-args) for
+usage details.
+
+<details><summary>Consider this solidity contract</summary>
```

Comment:
Is `details` and `summary` appropriate here? I

![image](https://user-images.githubusercontent.com/187813/207935561-d1033475-7f63-4360-9779-d34eb13d4d50.png)

Reply:
I didn't follow a style guide. I like the collapsible assets because it's easier to navigate the doc, and I'm curious what others think.

Reply:
I'm not a fan, personally.

Reply:
Is this where we call in the @OnlyOneJMJQ ?

Reply:
I like collapsible blocks, but they should provide additional information on top of essential snippets included in the body. So here there would be some minimal snippet above, perhaps a truncated import statement and one-line usage.

Of course they need to some styling adjustments as well, but we could probably just get away with `primary blue -> truffle green`, and `background light blue -> white` for now.

Reply:
I'm okay with the collapsed details/summary usage in the updated version. I think the word "Example" is better than "Click to see full sample files.", but it's not a show stopped for me.

The only issue is that the code's font within the expanded details is kind of small compared to the rest of the page:

![image](https://user-images.githubusercontent.com/187813/210664245-bdd05c04-ea31-46f7-ad9a-e5bb37fc1a0e.png)